l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py

```diff
@@ -97,3 +97,21 @@ def test_05_export_success(self):
         record.name += '%s01' % os.sep
         record.button_generate()
         self.assertTrue(record)
+
+    def test_06_include_moves_from_inactive_journals(self):
+        ''' Include moves off of inactive journals '''
+        record = self.env['xaf.auditfile.export'].create({})
+        record.button_generate()
+        self.assertTrue(record)
+        move_count = record.get_move_line_count()
+        all_journals = record.get_journals()
+
+        all_journals.write({'active': False})
+        self.assertTrue(all(not j.active for j in all_journals))
+
+        record_after = self.env['xaf.auditfile.export'].create({})
+        record_after.button_generate()
+        self.assertTrue(record_after)
+
+        move_count_after = record_after.get_move_line_count()
```

Comment:
I'm afraid this test is not meaningful for the bug, because `get_move_line_count` uses SQL and does not even touch the journal dimension. Maybe you can parse the document using etree and count the number of transaction nodes?

Reply:
Thanks for the suggestion, modified it in such a way! 🙏